TappNetwork/home-visiting-app-web
# Cu 2pz9np2 page refresh bugs

# Details

Fixes for page refresh errors. All pages within the app can now be hard refreshed without crashing the app, and the sidebar now shows current page even when refreshed. Some minor code cleanup is also included. 

# Steps to test changes

1. visit /login with visitor credentials
2. Refresh page on a few different pages and see that sidebar still shows current page
3. Go to a supervisor profile page and a program profile page and refresh. 
4. Go to edit profile page and refresh.
5. Ensure latest backend master branch is pulled down and submit a change to profile with an empty select many field. These changes should be reflected on the profile page upon a refresh.
6. Navigate to dashboard, click description cards and ensure sidebar shows correct page after redirect
7. resize page and ensure main header text is never truncated 

# Issues Addressed

[3887775](https://app.clickup.com/t/3887775)

# Screenshots

Any visual changes must have screenshots
